examples/09_cssmodules/src/index.html

```diff
@@ -22,15 +22,15 @@
       #root > .spinner {
         margin-top: calc(50% - 18px);
       }
-    </style>
+    </style> -->
   </head>
   <body>
     <!--placeholder1-->
     <div id="root">
       <div class="spinner"></div>
     </div>
     <!--/placeholder1-->
-    <script src="/main.tsx" async type="module"></script>
+    <script src="/src/main.tsx" async type="module"></script>
```

Comment:
I see what you mean. This is probably unexpected to normal vite users.
How about doing
```suggestion
    <script src="main.tsx" async type="module"></script>
```
and make it an absolute path with a new vite plugin for both DEV and PRD?

But, actually, I don't think it's super important. People don't often modify this file from the template.
Yeah, I think `"/src/main.tsx"` is acceptable on second thought.